src/current/v24.1/known-limitations.md

```diff
@@ -6,699 +6,624 @@ keywords: gin, gin index, gin indexes, inverted index, inverted indexes, acceler
+~~~
+     ?column?
+------------------
+  string1string2
+(1 row)
+~~~
 
-[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/93558)
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SELECT ('string1' collate en) || ('string2' collate en);
+~~~
 
-### Limitations for `DROP OWNED BY`
+~~~
+pq: unsupported binary operator: <collatedstring{en}> || <collatedstring{en}>
+~~~
 
-[`DROP OWNED BY`]({% link {{ page.version.version }}/drop-owned-by.md %}) drops all owned objects as well as any [grants]({% link {{ page.version.version }}/grant.md %}) on objects not owned by the [role]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles).
+[Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/10679)
 
-#### `DROP OWNED BY` is not supported where role has system-level privileges
+#### Current sequence value not checked when updating min/max value
 
-{% include {{page.version.version}}/known-limitations/drop-owned-by-role-limitations.md %}
+Altering the minimum or maximum value of a series does not check the current value of a series. This means that it is possible to silently set the maximum to a value less than, or a minimum value greater than, the current value. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/23719)
 
-### Spatial features disabled for ARM Macs
+#### `null_ordered_last` does not produce correct results with tuples
 
-[Spatial features]({% link {{ page.version.version }}/spatial-data-overview.md %}) are disabled due to an issue with macOS code signing for the [GEOS](https://libgeos.org/) libraries. Users needing spatial features on an ARM Mac may instead [use Rosetta](https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta) to [run the Intel binary]({% link {{ page.version.version }}/install-cockroachdb-mac.md %}#install-binary) or use the [Docker image]({% link {{ page.version.version }}/install-cockroachdb-mac.md %}#use-docker) distribution. This is expected to be resolved in an upcoming 22.2 patch release.
+By default, CockroachDB orders `NULL`s before all other values. For compatibility with PostgreSQL, the `null_ordered_last` [session variable]({% link {{ page.version.version }}/set-vars.md %}) was added, which changes the default to order `NULL`s after all other values. This works in most cases, due to some transformations CockroachDB makes in the optimizer to add extra ordering columns. However, it is broken when the ordering column is a tuple. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/93558)
 
-[GitHub tracking issue](https://github.com/cockroachdb/cockroach/issues/93161)
+### Transactions
 
-### Limited SQL cursor support
+#### Read Committed features
```

Comment:
It seems like you could combine this with the next under the single heading "Read Committed".